docs/ref/contrib/admin/index.txt

```diff
@@ -2749,6 +2749,61 @@ The presence of the ``admin_password_reset`` named URL will cause a "forgotten
 your password?" link to appear on the default admin log-in page under the
 password box.
 
+.. _admin-logentries:
```

Comment:
You can omit this since it's unused and we'd probably prefer linking to the LogEntry class anyway.